app/components/Application/__tests__/index.js

```diff
@@ -3,8 +3,8 @@ import ReactTestUtils from 'react-addons-test-utils';
 import Application from '../index.jsx';
 import styles from '../style.sass';
 
-describe('Application', function() {
-  it('displays the component', function() {
+describe('Application', function () {
+  it('displays the component', function () {
```

Comment:
Can you please remove these leading spaces before each function so that eslint passes? There are a few more throughout the PR too.